ui/settings_page.go

```diff
@@ -2,36 +2,357 @@ package ui
 
 import (
 	"gioui.org/layout"
+	"gioui.org/widget"
 
 	"github.com/planetdecred/godcr/ui/decredmaterial"
+	"github.com/planetdecred/godcr/ui/values"
+	"github.com/planetdecred/godcr/wallet"
 )
 
-const PageSettings = "SettingsPage"
+const PageSettings = "Settings"
 
 type settingsPage struct {
-	dummyText decredmaterial.Label
+	pageContainer layout.List
+	theme         *decredmaterial.Theme
+	walletInfo    *wallet.MultiWalletInfo
+	wal           *wallet.Wallet
+
+	currencyConversion decredmaterial.IconButton
+	connectToPeer      decredmaterial.IconButton
+	userAgent          decredmaterial.IconButton
+	changeStartupPass  decredmaterial.IconButton
+
+	spendUnconfirm  *widget.Bool
+	startupPassword *widget.Bool
+	notificationW   *widget.Bool
```

Comment:
There doesn't seem to be any other field named "notification". This is fine as just "notification"

Reply:
There is a `notification()`